confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -104,6 +114,10 @@ def _cache_schema(self, schema, schema_id, subject=None, version=None):
                 self._add_to_cache(self.subject_to_schema_versions,
                                    subject, schema, version)
 
+    def _cache_json_schema(self, schema, schema_id):
+        if schema_id not in self.id_to_schema:
```

Comment:
shouldn't this be: if schema_id not in self.id_to_json_schema

I don't see how json schema is getting cached as _cache_json_schema gets called after _cache_schema in register